examples/displayer.rs

```diff
@@ -0,0 +1,324 @@
+use anyhow::Result;
+use scylla::{
+    client::{session::Session, session_builder::SessionBuilder},
+    response::query_result::{ByteDisplaying, QueryRowsResult},
+    DeserializeRow,
+};
+use std::env;
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    let uri = env::var("SCYLLA_URI").unwrap_or_else(|_| "127.0.0.1:9042".to_string());
+
+    println!("Connecting to {} ...", uri);
+
+    let session: Session = SessionBuilder::new().known_node(uri).build().await?;
+
+    session.query_unpaged("CREATE KEYSPACE IF NOT EXISTS examples_ks WITH REPLICATION = {'class' : 'NetworkTopologyStrategy', 'replication_factor' : 1}", &[]).await?;
+
+    session
+        .query_unpaged(
+            "CREATE TABLE IF NOT EXISTS examples_ks.basic (a int, b int, c text, primary key (a, b))",
+            &[],
+        )
+        .await?;
+
+    session
+        .query_unpaged(
+            "INSERT INTO examples_ks.basic (a, b, c) VALUES (?, ?, ?)",
+            (3, 4, "lorem Ipsum jest tekstem stosowanym jako przykładowy wypełniacz w przemyśle poligraficznym. Został po raz pierwszy użyty w XV w. przez nieznanego drukarza do wypełnienia tekstem próbnej książki. Pięć wieków później zaczął być używany przemyśle elektronicznym,"),
+        )
+        .await?;
+
+    session
+        .query_unpaged(
+            "INSERT INTO examples_ks.basic (a, b, c) VALUES (1, 2, 'abc')",
+            &[],
+        )
+        .await?;
+
+    let prepared = session
+        .prepare("INSERT INTO examples_ks.basic (a, b, c) VALUES (?, 7, ?)")
+        .await?;
+    session
+        .execute_unpaged(&prepared, (42_i32, "I'm prepared!"))
+        .await?;
+    session
+        .execute_unpaged(&prepared, (43_i32, "I'm prepared 2!"))
+        .await?;
+    session
+        .execute_unpaged(&prepared, (44_i32, "I'm prepared 3!"))
+        .await?;
+
+    // Or as custom structs that derive DeserializeRow
+    #[allow(unused)]
+    #[derive(Debug, DeserializeRow)]
+    struct RowData {
+        a: i32,
+        b: Option<i32>,
+        c: String,
+    }
+
+    // example 1 - basic table
+    let result: QueryRowsResult = session
+        .query_unpaged("SELECT a, b, c FROM examples_ks.basic", &[])
+        .await?
+        .into_rows_result()?;
```

Comment:
:wrench:  It's an anti-pattern to perform `SELECT`s with `*_unpaged` methods.

:thought_balloon: :pushpin:  Unfortunately, only `*_single_page` methods fulfil the following requirement:
1. they are recommended for `SELECTs`, **and**
2. they return `QueryResult`.

`*_iter` methods return `QueryPager`, which is a distinct type that `QueryResult`.
I think it would be beneficial to support displaying of rows coming through both `QueryResult` and `QueryPager`. Extending the framework for `QueryPager`, however, can be left for a follow-up.

:wrench: For now, let's use `query_single_page` in a loop here.